package.json

```diff
@@ -199,7 +199,7 @@
   ],
   "scripts": {
     "vscode:prepublish": "npm run esbuild-base -- --minify",
-    "esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=dist/extension.js --external:vscode --format=cjs --platform=node --target=node14",
+    "esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=dist/extension.js --external:vscode --format=cjs --platform=node",
```

Comment:
Should this be changed to `--target=node16` instead of just removing it?

Reply:
We could keep it, but would need to continually monitor it.  I believe that the default is the current active branch, which seems like the better approach to me.

Reply:
Without the `--target` are we not building with whatever is setup on the computer. We could see changes based on who builds the package. 

I'm not overly experienced here so don't know if this is a real issue.

Reply:
Ah, interesting.  Okay, then I suppose it makes sense to use `--target=node16`.  However, it does mean that we would need to be a bit more vigilant to keep it up-to-date.